Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -940,6 +940,9 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
 #if ENABLE(GPU_PROCESS)
 void WebPage::gpuProcessConnectionDidBecomeAvailable(GPUProcessConnection& gpuProcessConnection)
 {
+#if ENABLE(ALTERNATE_WEBM_PLAYER)
+    gpuProcessConnection.setAlternateWebMPlayerEnabled(DeprecatedGlobalSettings::isAlternateWebMPlayerEnabled());
```

Comment:
There might be a better place to put this update rather than in the `gpuProcessConnectionDidBecomeAvailable` event?